app/assets/javascripts/admin/forms/forms.js.coffee

```diff
@@ -0,0 +1,25 @@
+$ ->
+  $('.edit-action').click () ->
+    $('.show-view').addClass('edit-view').removeClass('show-view')
+
+  $('.show-action').click () ->
+    $('.edit-view').addClass('show-view').removeClass('edit-view')
+
+  $('.update-action').click (e) ->
+    e.preventDefault()
+
+    form = $(this).closest('form')
+    id = $(form).attr('data-id')
+    $(form).attr('method', 'put')
+    data = $(".organization-record").serialize()
+    location = window.location.href
+
+    $.ajax
+      method: 'PUT',
+      url: location,
+      data: data,
+      success: (data) ->
+        console.log("Updated")
+        window.location.href = data
+      error: (data) ->
+        console.log(data)
```

Comment:
Errors should be loading on server side with Simple Form and alerts. This console should not be here.